templates/base.yaml

```diff
@@ -53,6 +53,14 @@ Resources:
               - !Ref DNSDomainName
           ValidationDomain: !Ref DNSDomainName
       ValidationMethod: DNS
+  SSLCertificateSageIT:
```

Comment:
i see, we don't want two instances of this if we deploy multiple login apps to the same account. in that case i suggest we move this into a new separate extras.yaml file so that it's decoupled with the rest of the resources.  that will make it easier to manage going forward.

Reply:
That's what I thought. Will do.